### DISCUSSION TITLE
Examples of Link? Where is it supposed to be used?

### DISCUSSION THREAD

**Discussion by nightroman** [Megabit/Blazorise#2319] [Q&A] [answered]
Are there any examples of `Link`? I cannot see any in Blazorise.Demo.

Why I am asking. I am trying something like this:

```xml
<TableRowCell><Link To="@it.HomepageUrl" Target="Target.None">@it.HomepageUrl"</Link></TableRowCell>
```

It does not compile, complains about malformed markup.

The workaround is using `a`. But I am curious why `Link` is not working.

```xml
<TableRowCell><a href="@it.HomepageUrl" target="_blank">@it.HomepageUrl</a></TableRowCell>
```

**Comment by stsrki** [comment, marked as answer]
It's a bug in VS that is still not fixed. See here https://github.com/stsrki/Blazorise/issues/1353

**Reply by nightroman** [reply]
I see. All right.

> I will leave it as a Link but I will advise it to the users of my library to always write it as Blazorise.Link. So once the issue is fixed I will just remove the note from my documentation.

If the fix is not coming soon, mentioning the workaround in https://blazorise.com/docs/components/link/ would help.
